vm_helper_files/Makefile

```diff
@@ -100,11 +100,12 @@ install-release-online:
 		elif grep -q -i "release 6" /etc/redhat-release; then \
 			yum install -y https://github.com/quipucords/qpc-tools/releases/download/$(tools_version)/qpc-tools.el6.noarch.rpm; \
 		else \
+			yum install -y epel-release; \
```

Comment:
This works 🎉! Is there any reason we can't call `make setup` in install-release-online?

Reply:
Great suggestion Ash 🌮